ts/features/zendesk/screens/ZendeskAskPermissions.tsx

```diff
@@ -179,13 +185,13 @@ const ZendeskAskPermissions = () => {
     workUnitCompleted();
   };
 
-  // TODO: open directly a ticket with category "payment" if assistanceForPayment: https://pagopa.atlassian.net/browse/IA-575
   const handleOnContinuePress = () => {
     // if is not possible to get the config or if the config has any category open directly a ticket.
     if (
       !isReady(zendeskConfig) ||
       Object.keys(zendeskConfig.value.zendeskCategories?.categories ?? {})
-        .length === 0
+        .length === 0 ||
+      assistanceForPayment
```

Comment:
Could you update the comment since we're adding a new condition?
Maybe a more readable code could be the adding of a dedicated function (skip this suggestion if you don't like it)

```typescript
const canSkipCategoryChoice = (): boolean =>
    !isReady(zendeskConfig) ||
    Object.keys(zendeskConfig.value.zendeskCategories?.categories ?? {})
      .length === 0 ||
    assistanceForPayment;
```